README.md

```diff
@@ -13,21 +13,26 @@ Visit [nix.dev](https://nix.dev) for [installation instructions](https://nix.dev
 
 Full reference documentation can be found in the [Nix manual](https://nixos.org/nix/manual).
 
-## Building And Developing
+## Building and developing
 
-See our [Hacking guide](https://nixos.org/manual/nix/unstable/contributing/hacking.html) in our manual for instruction on how to
- set up a development environment and build Nix from source.
+See our [Hacking guide](https://nixos.org/manual/nix/unstable/contributing/hacking.html) for instruction on how to set up a development environment and build Nix from source.
 
 ## Contributing
 
 Check the [contributing guide](./CONTRIBUTING.md) if you want to get involved with developing Nix.
 
-## Additional Resources
+## Additional resources
 
-- [Nix manual](https://nixos.org/nix/manual)
-- [Nix jobsets on hydra.nixos.org](https://hydra.nixos.org/project/nix)
-- [NixOS Discourse](https://discourse.nixos.org/)
-- [Matrix - #nix:nixos.org](https://matrix.to/#/#nix:nixos.org)
+Nix was created by Eelco Dolstra and developed as the subject of his PhD thesis [The Purely Functional Software Deployment Model](https://edolstra.github.io/pubs/phd-thesis.pdf), published 2006.
+
+Today, a world-wide developer community contributes to Nix and the ecosystem that has grown around it.
```

Comment:
This doesn't need to be in the README. (Maybe the manual could have a "history" section or something like that.)

Reply:
I have the other PR that reworks the introduction. After letting it sink in a bit, I'm see two problems with my set of changes:
- information is duplicated and there's no clean mechanism against that (partial file inclusion into the manual is brittle), so that background stuff (which I think is important) should live on one place
- that one place may not be the manual. in the other PR I tried to write a purpose statement for the manual, and that is looking up facts and details about interfaces and behavior. lore is not that. IMO we need a place for the lore, and it should be somewhere early on in the user journey, because it's essential in forming a mental model about non-technical aspects of the project.

Since we can reasonably expect the readme to be one of the first points of contact with the project itself (rather with just the artifact), I suggest moving more lore here. Also because that's what's tends to change or be revised, while the manual is attached to a particular nix release and therefore frozen in time.

Reply:
tl;dr the manual should probably be reference-only, a bit of up-to-date of entry-point context in the README seems okay.

Objections?